core/state_processor_test.go

```diff
@@ -44,6 +44,12 @@ func (c MockChainContext) GetHeader(common.Hash, uint64) *types.Header {
 	return c.blocks[1].Header()
 }
 
+/*
+if parentOfParent == nil {
```

Comment:
🚓 🚨 cruft police pulled you over and issued a citation for littering the codebase with dead code

Reply:
fixed